chrome/content/zotero/itemTree.jsx

```diff
@@ -59,7 +58,9 @@ var ItemTree = class ItemTree extends LibraryTree {
 		dragAndDrop: false,
 		persistColumns: false,
 		columnPicker: false,
-		columns: COLUMNS,
+		columns: Zotero.ItemTreeManager.columns,
+		extraColumns: [],
+		columnsFilter: (columns) => columns,
```

Comment:
This doesn't work, because we might want to create an ItemTree with some of the default columns, with custom properties for width, visibility, etc. 

Generally it doesn't really address the issue of multiple sources of truth. In thinking more, it's unlikely we can have a single source of truth regarding columns anyway. What if some plugin creates a column "relationNumber", which displays the number of related items, and then we create some view, where we have that column (but it's not in the main view). Which one should take precedence? It's probably the one that was specified during tree init.

I don't think we can avoid just having a `columns` prop here, and then any registration of custom columns that use the same dataKeys should not be displayed in that itemTree and the registrator should be notified.

Also, the columns in the `itemTreeColumns.js` file should not be treated in a special way. It's a list of columns that may appear in an ItemTree in Zotero, but not all of them have to appear, nor is it an exhaustive list of columns that may appear.